pkgs/by-name/sw/switcheroo/package.nix

```diff
@@ -48,11 +49,18 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     glib
     gtk4
+    imagemagick
     libadwaita
   ] ++ lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.Foundation
   ];
 
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix PATH : "${imagemagick}/bin"
```

Comment:
```suggestion
      --prefix PATH : "${lib.makeBinPath [ imagemagick ]}"
```

Reply:
Thanks, I did not know about that function :smile: